src/Behat/Testwork/EventDispatcher/TestworkEventDispatcher.php

```diff
@@ -17,8 +17,27 @@
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
  */
+$isSymfony5AndPhpGreater72 = function () {
+    $reflection  = new \ReflectionClass(\Symfony\Component\EventDispatcher\EventDispatcher::class);
+    $dispatch    = $reflection->getMethod('dispatch');
+    $first_param = $dispatch->getParameters()[0];
+    switch ($first_param->getName()) {
+        case 'event':
+            // This is the new Symfony 5 event dispatcher interface
+            // This test retained from existing code. In reality it's unclear that it could ever be false, since
+            // symfony/event-dispatcher 5.x requires php: ^7.2.5 in their composer, and before that the class has the
+            // old signature.
+            return PHP_VERSION_ID > 70200;
```

Comment:
I have left this PHP > 7.2 check in for now, as I'm not clear quite why it was added originally. It's not possible to have symfony 5 on an older php version so I think it was always redundant.

Reply:
You're correct, this could be removed

Reply:
Removed the 7.2 check, can squash it all up once we have a solution to the global lambda